include/eve/module/core/regular/fma.hpp

```diff
@@ -13,31 +13,28 @@
 
 namespace eve
 {
-template<typename Options>
-struct fma_t : strict_elementwise_callable<fma_t, Options, pedantic_option, promote_option>
-{
-  template<eve::value T,eve::value U,eve::value V>
-  requires(Options::contains(promote))
-  constexpr EVE_FORCEINLINE auto operator()(T a, U b, V c) const noexcept { return EVE_DISPATCH_CALL(a,b,c); }
+  template<typename Options>
+  struct fma_t : strict_elementwise_callable<fma_t, Options, pedantic_option, promote_option>
+  {
+    template<eve::value T,eve::value U,eve::value V>
+    requires(Options::contains(promote))
+      constexpr EVE_FORCEINLINE auto operator()(T a, U b, V c) const noexcept { return EVE_DISPATCH_CALL(a,b,c); }
```

Comment:
why the inconsistent tabulation ? Don't change code for no reason.

Reply:
done